main.cpp

```diff
@@ -752,6 +910,8 @@ struct Computer
     void increaseScreenBrightness(Display screen, double numBrightness = 0.5);
     // increase number of core processing for CPU
     bool increaseCoreProcessing(CPU currentCpu, int numCores = 1);
+    // new member function
+    Computer purchaseNewComputer(int threshold, int startingValue);
```

Comment:
a Computer instance will return another Computer?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Computer instance...
- [ ] revise